test/unit/behaviours/get-declarer.test.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const reqres = require('reqres');
```

Comment:
it would be good to understand reqres more, it confuses me, even though I got it working